03-specifying-sample-designs.Rmd

```diff
@@ -435,7 +442,7 @@ The different types of replicate weights largely differ in step 1 - how the samp
 
```

Comment:
@szimmer: you mentioned you wanted to add some information about scales here.

Reply:
@szimmer would you like to add this?